src/contract/ContractEditor.tsx

```diff
@@ -369,19 +370,27 @@ const ContractEditor = observer(
       onReset()
     }, [contract, onReset])
 
-    let [removeContract, { loading: removeLoading }] = useMutationData(removeContractMutation)
+    let [removeContract, { loading: removeLoading }] = useMutationData<ContractType>(
+      removeContractMutation
+    )
```

Comment:
```suggestion
    let [removeContract, { loading: removeLoading }] = useMutationData<ContractType>(
      removeContractMutation,
      {
        refetchQueries: [
          {
            query: contractsQuery,
            variables: {
              operatorId: contract.operatorId,
            },
          },
        ],
      }
    )
```

You can use refetchQueries to automatically refetch some queries after a mutation. This way you do not need to use `no-cache`.